### DISCUSSION TITLE
「委托」———委托方天才俱乐部#76席，螺丝咕姆

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#2458] [其他]
- 以机械贵族螺丝咕姆一世的名义，清除诸界疯狂失控的机械生命体。

「帝皇鲁珀特」
一切有机生命的大政！它曾赋予众机之灵一个宏伟的任务：净空一切有机生命，将繁星铸造成精密运转的机械一一吾在此请求，请將数宇使命禁锢的机器彻底解放。
一一螺丝咕姆，天才俱乐部#76

**Comment by misaka-lastorder** [comment]
这个委托我接了

**Comment by RSsuzuki** [comment]
这个委托我接了
/
